ambry-tools/src/main/java/com/github/ambry/tools/admin/ServerAdminTool.java

```diff
@@ -643,6 +656,29 @@ public Pair<ServerErrorCode, BlobAll> getAll(DataNodeId dataNodeId, BlobId blobI
     return new Pair<>(response.getFirst(), blobAll);
   }
 
+  /**
+   * Get the blob index values for {@code blobId}
+   * @param dataNodeId the {@link DataNodeId} to contact.
+   * @param blobId the {@link BlobId} to operate on.
+   * @return the {@link ServerErrorCode} and the json serialized content of blob index values.
+   * @throws Exception
+   */
+  public Pair<ServerErrorCode, String> getBlobIndex(DataNodeId dataNodeId, BlobId blobId) throws Exception {
+    AdminRequest adminRequest =
+        new AdminRequest(AdminRequestOrResponseType.BlobIndex, null, correlationId.incrementAndGet(), CLIENT_ID);
+    BlobIndexAdminRequest blobIndexAdminRequest = new BlobIndexAdminRequest(blobId, adminRequest);
+    ResponseInfo response = sendRequestGetResponse(dataNodeId, blobId.getPartition(), blobIndexAdminRequest);
+    AdminResponseWithContent adminResponse =
+        AdminResponseWithContent.readFrom(new NettyByteBufDataInputStream(response.content()));
+    response.release();
+    ServerErrorCode errorCode = adminResponse.getError();
+    String content = "";
```

Comment:
What's the empty string as a Json format? Can Json parser parse "" without error? 
Or shall we have it as "{}"?

Reply:
there will no json parser decoding the content. I will just print out the content to whoever is using this tool.